.travis.yml

```diff
@@ -5,7 +5,7 @@ language: generic
 env:
   REPO: docksal/ssh-agent
   IMAGE_DNS: ${REPO}:dev
-  DOCKSAL_VERSION: develop
+  DOCKSAL_VERSION: feature/ssh-key
```

Comment:
@lmakarov  should it be like this?

Reply:
Yes. 1.2 depend on the yet to be merged functionality in fin. Once merged into develop, we can switch it back.